Joanna

Joanna-G opened the pull request.

Joanna-G committed:
GUI Files

I know we haven't talked about this yet so I hope I haven't stepped on any toes. This doesn't have to stay, I was just wanting to do something fun and I know I have only two classes and they're pretty easy. This is just some bare bones stuff I was playing with.

Joanna-G committed:
Rename GUI.java to ChargerMail/src/GUI.java

Joanna-G committed:
Rename GUITester.java to ChargerMail/src/GUITester.java

t-walker-21 reviewed (commented):
Have you tested this code?

Joanna-G commented:
It runs. It doesn't really do anything though. I forgot that Komlan said he'd do it, so we can just remove these.

Joanna-G committed:
Update Mailbox.java

Added a name for mailboxes and an empty trash method.

Joanna-G committed:
Update Mailbox.java

More modifications. Added some comments where discussion needs to be had about how the classes communicate. Not exactly sure how to handle a multi-line message, so for now I'm using a String ArrayList to hold lines.

Joanna-G committed:
Update Message.java

Joanna-G committed:
Delete GUI.java

Joanna-G committed:
Delete GUITester.java
